storage/rocksdb/ha_rocksdb.cc

```diff
@@ -2474,13 +2472,6 @@ static int rocksdb_prepare(handlerton *const hton, THD *const thd,
       if (!tx->prepare(rdb_xid_to_string(xid))) {
         return HA_EXIT_FAILURE;
       }
-      if (thd->durability_property == HA_IGNORE_DURABILITY &&
```

Comment:
No I can not, this was part of the original upstream behavior/patch that integrated with Facebook MySQL group commit, which is similar to what is implemented in PS 5.7 but missing from PS 5.6, see https://github.com/facebook/mysql-5.6/commit/14a0d4a97c09b52fa7450e6a3d56ebe7ed193ab6

It may not be required at all for the purpose of PS 5.6